code/modules/surgery/organs/lungs.dm

```diff
@@ -298,15 +298,12 @@
 	// Nitryl
 		var/nitryl_pp = breath.get_breath_partial_pressure(breath_gases[/datum/gas/nitryl][MOLES])
 		if (prob(nitryl_pp))
-			to_chat(H, "<span class='alert'>Your mouth feels like it's burning!</span>")
-		if (nitryl_pp >40)
-			H.emote("gasp")
-			H.adjustFireLoss(10)
-			if (prob(nitryl_pp/2))
-				to_chat(H, "<span class='alert'>Your throat closes up!</span>")
-				H.silent = max(H.silent, 3)
+			H.emote("burp")
+		if (nitryl_pp>40)
+			H.adjustOrganLoss(ORGAN_SLOT_LUNGS, 25)
+			to_chat(H, "<span class='notice'>You feel a burning sensation in your throat</span>")
 		else
-			H.adjustFireLoss(nitryl_pp/4)
+			H.adjustOrganLoss(ORGAN_SLOT_LUNGS, nitryl_pp/2)
```

Comment:
it's like 2am or something but 40/2 is 20, shouldn't this be a smooth curve? also the burning thing should prob be where burps are, cause 39% nitryl is still pretty bad